www/source/partials/docs/_dev-pkgs-share.html.md.erb

```diff
@@ -67,18 +67,18 @@ For more information on how to use channels, see [Continuous Deployment Using Ch
 
 > **Note:** When running private packages from Builder, it's necessary to add your [Habitat access token](/docs/using-builder/#builder-token) to the machine where you intend to deploy the package, via `export HAB_AUTH_TOKEN=<token>`.
 
-You can instruct the Supervisor to download and run packages from Builder by using the `hab start` command, for example:
+You can instruct the Supervisor to download and run packages from Builder by using the `hab sup run` command, for example:
```

Comment:
This (and the next two) need to be fixed. See https://www.habitat.sh/docs/using-builder/#sharing-pkgs